health/notifications/health_alarm_notify.conf

```diff
@@ -212,8 +212,27 @@ DEFAULT_RECIPIENT_EMAIL="root"
 # to corelate messages and also will cause most clients to group all the
 # messages together.  This is enabled by default, uncomment the line
 # below if you want to disable it.
-#EMAIL_THREADING="NO"
+#EMAIL_THREADING="YES"
```

Comment:
This value was and is commented-out, so no need to change it.

Reply:
I put it back to default value. I made some tests before and forgot about this lite line